src/PostSharp.Engineering.BuildTools/Build/Model/Product.cs

```diff
@@ -1348,6 +1354,7 @@ void AppendToArcadeSuffix( string s )
         <{this.ProductNameWithoutDot}VersionFilePath>{this.VersionsFilePath}</{this.ProductNameWithoutDot}VersionFilePath>
         <{this.ProductNameWithoutDot}BuildNumber>{buildSettings.BuildNumber}</{this.ProductNameWithoutDot}BuildNumber>
         <{this.ProductNameWithoutDot}BuildType>{buildSettings.BuildType}</{this.ProductNameWithoutDot}BuildType>
+        <{this.ProductNameWithoutDot}BuildDate>{buildDate}</{this.ProductNameWithoutDot}BuildDate>
         <{this.ProductNameWithoutDot}ArtifactsDirectory>$(MSBuildThisFileDirectory)</{this.ProductNameWithoutDot}ArtifactsDirectory>
```

Comment:
It's good it's there but it shoudl also be in the `Versions.(Config).props` file under the name `BuildDate` which does not depend on the package being built. This property is then consumed by `AssemblyMetadata.targets` without possibility of user error.